sycl/doc/extensions/experimental/sycl_ext_oneapi_graph.asciidoc

```diff
@@ -245,15 +271,60 @@ represents either a command-group or an empty operation.
 through newly added interfaces. This is either using the `make_edge()` function
 to define an edge between existing nodes, or using a
 `property::node::depends_on` property list when adding a new node to the graph.
+Nodes or <<event-terminology, graph-limited events>> passed to this property
+that are from the same graph will create edges within the graph.
```

Comment:
Where does the `graph-limited event of the same graph` come from in the context of 'Explicit Graph Building API'?

Reply:
You can mix and match APIs, so you could record a node into a graph, and then use the explicit API to add further nodes that depend on that recorded node.

Reply:
imo, the `add` function for explicit API does not return event, while the `get_event` function is the member of `command_graph<graph_state::executable>`, we are unable to get an event of `the same graph` in the context of 'Explicit Graph Building API'

could you share some code? thanks.

Reply:
I've talked about it before but was putting off adding it to this PR since it's already quite large, but I've now added the `node::get_event_from_node()` function which is a counterpart to the existing `node::get_node_from_event()` function.

With these you can take a `node` recorded from the explicit API and obtain the graph-limited event for it (as if it had been recorded via queue recording), and vice-versa to allow you to mix API usage.